mmdeploy/utils/utils.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+
+from mmcv.utils import get_logger
+
+
+def get_root_logger(log_file=None, log_level=logging.INFO) -> logging.Logger:
+    """Get root logger.
+
+    Args:
+        log_file (str, optional): File path of log. Defaults to None.
+        log_level (int, optional): The level of logger.
+            Defaults to logging.INFO.
+    Returns:
+        :obj:`logging.Logger`: The obtained logger
```

Comment:
could we make this docstring consistent with the codebase?

Reply:
It is consistent with mmdetection.... Shall we make an adaption?

Reply:
We could change the format in another PR if necessary.